index.html

```diff
@@ -577,6 +578,30 @@ <h2 id="shipment_status">shipment_status</h2>
                         </td>
                     </tr>
                 </table>
+                <h2 id="main_status">main_status</h2>
```

Comment:
Het heet `main` in de api toch?

```suggestion
                <h2 id="main">main</h2>
```